.github/workflows/update.yml

```diff
@@ -0,0 +1,42 @@
+name: Prepare for FHEM update
+
+on: [push]
+
+jobs:
+  update:
+    env:
+      CONTROLS_FILENAME: controls_Buienradar.txt
+      CONTROLS_CREATE_SCRIPT: build/prepare_controls.sh
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update Environment
+        run: sudo apt-get install -y pandoc tidy
+      - name: Extract branch name
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        id: extract_branch
+      - name: Checkout Repository
+        uses: actions/checkout@master
+      - name: update controls files
+        uses: fhem/fhem-controls-actions@master
+        with:
+          filename: $CONTROLS_FILENAME
+      - name: Update Controls
```

Comment:
Das brauchst Du vermutlich nicht mehr oder, weil Du ja die GitHub Action dafür eingebaut hast :)

Reply:
Das ist die Config für die Github Action ;-)

Reply:
De Abschnitt update controls files erzeugt das neue Control File.

Dein Script in diesem Abschnitt macht ja noch mal das gleiche oder?